qiskit/providers/models/backendconfiguration.py

```diff
@@ -610,3 +685,38 @@ def describe(self, channel: ControlChannel) -> Dict[DriveChannel, complex]:
         for u_chan_lo in self.u_channel_lo[channel.index]:
             result[DriveChannel(u_chan_lo.q)] = u_chan_lo.scale
         return result
+
+    def _parse_channels(self, channels: Dict[set, Any]) -> Dict[Any, Any]:
+        r"""
+        Generates a dictionaries of ``Channel``\s, and tuple of qubit(s) they operate on.
+
+        Args:
+            channels: An optional dictionary containing information of each channel -- their
+                purpose, type, and qubits operated on.
+
+        Returns:
+            qubit_channel_map: Dictionary mapping tuple of qubit(s) to list of ``Channel``\s.
+            channel_qubit_map: Dictionary mapping ``Channel`` to list of qubit(s).
+            control_channels: Dictionary mapping tuple of qubit(s), to list of
+                ``ControlChannel``\s.
+        """
+        qubit_channel_map = defaultdict(list)
+        channel_qubit_map = defaultdict(list)
+        control_channels = defaultdict(list)
+        channel_prefix = {
+            'd': DriveChannel,
```

Comment:
Is it possible to use `DriveChannel.prefix` here instead? And similar for the others? If so, it might be better, because we could change `DriveChannel.prefix` and not need to remember to make any changes here

Reply:
yes, fixed in [0da2877](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/0da28775b838183eedd71d9157f20c08081e2833).

I'm leaving `'acquire'` as it is since `AcquireChannel.prefix` is `'a'` and in the backend, it's called `'acquire'`.